control_panel_api/tests/test_services.py

```diff
@@ -19,7 +19,7 @@
 )
 
 
-@patch.object(aws, 'client', MagicMock())
+@patch('control_panel_api.aws.aws.client')
```

Comment:
Just curious here, any reason why this changed? I see all tests now need to receive a `mock_client`.

Is it to make this more explicit that you're using a mock instead of calling `aws.client.return_value [...]`?

Reply:
I wrote in PR description.

Basically the MagicMock() in the patch is good for just mocking out a function and ignoring it.  However if you really want to mock and test the mocked function it's the wrong one as the mock is not cleared between test calls, so using the patch and inject is better.

Reply:
Oh I missed that sentence in the PR description 🤦‍♂️ 

Nice one, didn't know that (that's why I asked), so it's basically to avoid state leaking between tests, cheers 👍